amfora.1

```diff
@@ -0,0 +1,264 @@
+.El
+.Ss XDG
+.Bl -tag -width 2v -compact
+.It Pa $XDG_CONFIG_HOME/amfora/config.toml
+.It Pa $XDG_DATA_HOME/amfora/bookmarks.xml
+.El
+.Ss WINDOWS
+.Bl -tag -width 2v -compact
+.It Pa %APPDATA%\eamfora\econfig.toml
+.It Pa %APPDATA%\eamfora\ebookmarks.xml
+.El
+.Sh EXIT STATUS
+.Nm
+exits
+.Va 0
+on success, and
+.Va 1
+if an error occurs.
+.Sh EXAMPLES
+.Ss EXAMPLE BOOKMARK
+.Bd -literal
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE xbel
+  PUBLIC "+//IDN python.org//DTD XML Bookmark Exchange Language 1.1//EN//XML"
+         "http://www.python.org/topics/xml/dtds/xbel-1.1.dtd">
+<xbel version="1.1">
+    <bookmark href="gemini://example.com/">
+        <title>Example Bookmark</title>
+    </bookmark>
+</xbel>
+.Ed
+.Ss EXAMPLE PROXY
+.Bd -literal
+[proxies]
+# Allows setting a Gemini proxy for different schemes.
+# The settings are similar to the url-handlers section above.
+# E.g. to open a gopher page by connecting to a Gemini proxy server:
+#   gopher = "example.com:123"
+#
+# Port 1965 is assumed if no port is specified.
+#
+# NOTE: These settings override any external handlers specified in
+# the url-handlers section.
+#
+# Note that HTTP and HTTPS are treated as separate protocols here
+.Ed
+.Ss EXAMPLE CERTIFICATE
+.Bd -literal
+[auth]
+# Authentication settings
+
+[auth.certs]
+# Client certificates
+# Set domain name equal to path to client cert
+# "example.com" = "mycert.crt"
+"astrobotany.mozz.us" = "~/.local/share/amfora/astrobotany-cert.pem"
+
+[auth.keys]
+# Client certificate keys
+# Set domain name equal to path to key for the client cert above
+# "example.com" = "mycert.key"
+"astrobotany.mozz.us" = "~/.local/share/amfora/astrobotany-key.pem"
+.Ed
+.Sh DIAGNOSTICS
+Errors are printed to
+.Dv stderr .
+.Sh SEE ALSO
+The official documentation is maintained in two places:
+.Bl -tag -width 2v -compact
+.It Lk gemini://makeworld.space/amfora-wiki/
+.It Lk https://github.com/makeworld-the-better-one/amfora/wiki
+.El
+.Pp
+.Bl -tag -width 2v -compact
+.It agena.py gemini-to-gopher proxy.
+.It duckling-proxy.go gemini-to-HTTPS proxy.
+.El
+.Sh STANDARDS
+.Bl -tag -width 2v -compact
+.It Lk https://gemini.circumlunar.space/docs/specification.gmi
+Gemini Protocol
+.It Lk https://toml.io/en/
+Configuration syntax
+.It Lk http://pyxml.sourceforge.net/topics/xbel/docs/html/xbel.html
+Bookmark syntax
+.It Lk https://www.iana.org/assignments/media-types/media-types.xhtml
+Media Types
+.El
+.Sh HISTORY
+.Nm
+v1.0 was released on GitHub June 1, 2020 with tabs, TOFU, colors, history and input.
+Bookmarks, charsets, and search were added in v1.1 on Jun 24.
+Downloading was implemented in v1.3 on Jul 2. Themeing came in v1.4 on Jul 28.
+Emojies and extended proxy support came in v1.5 on Sep 01.
+Client certificates were implemented in v1.6 on Nov 4.
+Subscriptions were implemented in v1.7 on Dec 20, and a GitHub wiki was also started.
+v1.8 added media type handlers on Feb 17, 2021.
+v1.9 began storing bookmarks in XBEL on Dec 7.
```

Comment:
If you think this is necessary information, you might put in in a `.Bd -bullet` or something to separate each sentence onto separate lines.

Reply:
I think history is never necessary, but it always adds quality. Especially in the form of man pages, where this information is so condensed, it's really excellent. In my opinion, the quality/bloat factor of adding history to a man page is very good. Also, history is a standard section in the man page hierarchy.

 For some reason, I've never been able to make .Bl -bullet work, so I used .Bl -tag instead. Thanks again.